src/views/new evaluation/newevaluation.js

```diff
@@ -0,0 +1,529 @@
+            <input type="interviewer" placeholder="Interviewer">
+            <input type="Date" placeholder="Date">
+        </section>
+    `;
+}
+
+function technicalLevelTableHeader(options = {}) {
+    const headerElements = options.map((header) => {
+        return `<th>${header}</th>`;
+    });
+
+    return headerElements.join('');
+}
+
+function technicalLevelTableColumn(options = {}) {
+    const column = options.inputLevels.map((input) => `
+        <input type = "radio" name = "technicalLevel" value = "${input}">
+    `);
+    
+    return `
+        <td>
+            ${column.join('')}
+        </td>
+    `;
+}
+
+function technicalLevelTableBody(options = {}) {
+    const inputElements = options.map((items) => {
+        return technicalLevelTableColumn(items);
+    });
+    const row = inputElements.join('');
+    
+    return `
+        <tr class = "radioButtons">
+            ${inputElements}
+        </tr>
+    `;
+}
+
+function technicalLevel(options = {}) {
+    return `
+        <section>
+            <h2>${options.title}</h2>
+            <table>
+                <tr>
+                    ${technicalLevelTableHeader(options.headers)}
+                </tr>
+                ${technicalLevelTableBody(options.columnData)}
+            </table>
+        </section>
+    `;
+}
+
+function textarea(options = {}) {
+    return `
+        <section>
+            <h2>${options.header}</h2>
+            <textarea rows = "6" cols = "" placeholder = "${options.placeholder}"></textarea>
+        </section>
+    `;
+}
+
+function textareaSection(options = {}) {
+    return `
+        ${textarea({ header: 'Should the candidate be hired?', placeholder: 'The type of project that is suitable for the candidate. &#10; Is guidance required for the candidate', })}
+        ${textarea({ header: 'General Impression', placeholder: '*required', })}
+        ${textarea({
+            header: 'Workflow, Leadership &amp; Soft Skills', placeholder: 'Describe the environment in which the candidate works. &#10; Describe any guidance or management experience.',
+        })}
+    `;
+}
+
+function listItem(options = {}) {
+    const optionTag = options.map((el) => {
+        const optionElement = el.option.map((elem) => {
+            return `
+                <option>${elem}</option>
+            `;
+        });
+        
+        return `
+            <li>
+                <label>${el.label}</label>
+                <select name="${el.name}">
+                    <option disabled>Elements</option>
+                    ${optionElement.join('')}
+                </select>
+            </li>
+        `;
+    });
+
+    return optionTag.join('');
+}
+
+
+function fieldsetSection(options = {}) {
+    const fieldsetSectionElement = {
+        data: [{
+            legend: 'OOP, Design Patterns',
+            items: [{
+                label: 'Classes',
+                name: 'classes',
+                option: ['1', '2', '3']
```

Comment:
We could probably abstract away these `option` properties for now, such that we don't need to repeat them all over the place.